frontend/src/components/Profile/MyFriends.vue

```diff
@@ -25,14 +25,8 @@
 <script lang="ts">
 import axios from "axios";
 import Vue from "vue";
-
-interface ResponseFriendshipDto {
-  targetId: number;
-}
-
-interface PageDto<T> {
-  data: T[];
-}
+import { ResponseFriendshipDto } from "@dtos/friendships";
+import { PageDto } from "@dtos/pages";
```

Comment:
```suggestion
import { PageDto } from '@dtos/pages';
```

Reply:
Thank you, I had to rebase with master, they should have come from this.